community/index.md

```diff
@@ -31,6 +31,8 @@ The Scala Center focuses on education (especially online courses),
 documentation, open source community outreach, and tooling.  Community
 participation in all of these efforts is strongly encouraged.
 
+To receive security announcements or contact us about security issues, see our [security policy](/security/).
```

Comment:
The "Governance" section above would be a better place to add this (see https://www.scala-lang.org/community/), but there's currently no empty slot. Maybe "tooling summit" can move out...?

Reply:
Hmm... no super strong opinion here, but I think I would suggest adding a Security section near the bottom of the page. I don't think it really fits in Governance.